_codux/boards/toast/toast.board.tsx

```diff
@@ -0,0 +1,79 @@
+import { createBoard } from '@wixc3/react-board';
+import { Toast } from '~/src/components/toast/toast/toast';
+
+import styles from './toast.board.module.scss';
+
+export default createBoard({
+    name: 'Toasts',
+    Board: () => {
+        return (
+            <div className={styles.container}>
+                <Toast
+                    toastData={{
+                        type: 'success',
+                        id: '1',
+                        message: 'The product is added to the cart',
+                        pauseDuration: 0,
+                        ariaProps: {
+                            role: 'status',
+                            'aria-live': 'polite',
+                        },
```

Comment:
I wouldn't say I like having to pass things like this in the board. I expected it to be just `<Toast type="success" message="Hooray" />`. Anyway, I suggest at least extracting data such as `ariaProps`, `visible`, `createdAt` to a variable.

By the way, from what I see half of the data is not used in the Toast component. I'll ask again if we are not supposed to use `ToastBar` from `react-hot-toast`?

Reply:
> I wouldn't say I like having to pass things like this in the board

Me too.

> half of the data is not used in the Toast component

It is used in `resolveValue` function. 

> if we are not supposed to use ToastBar from react-hot-toast?

We can use `ToastBar` or custom render function. 
See `ToastBar` [implementation](https://github.com/timolins/react-hot-toast/blob/main/src/components/toast-bar.tsx). 
Using `ToastBar` I need to override all these styles:
```
const ToastBarBase = styled('div')`
  display: flex;
  align-items: center;
  background: #fff;
  color: #363636;
  line-height: 1.3;
  will-change: transform;
  box-shadow: 0 3px 10px rgba(0, 0, 0, 0.1), 0 3px 3px rgba(0, 0, 0, 0.05);
  max-width: 350px;
  pointer-events: auto;
  padding: 8px 10px;
  border-radius: 8px;
`;

const Message = styled('div')`
  display: flex;
  justify-content: center;
  margin: 4px 10px;
  color: inherit;
  flex: 1 1 auto;
  white-space: pre-line;
`;
```

And I don't know how to override `Message` styles, it is inner `div` in `ToastBar` without `className` prop.

Also I want user may drill into `Toast` component and edit elements and their styles in Codux what is impossible with third-party components.